pkg/proxy/ipvs/proxier.go

```diff
@@ -919,6 +947,11 @@ func (proxier *Proxier) syncProxyRules() {
 		klog.V(4).Infof("syncProxyRules took %v", time.Since(start))
 	}()
 
+	localAddrs, err := getLocalAddrs()
+	if err != nil {
+		klog.Errorf("Failed to get local addresses during proxy sync: %v, assuming external ip is not local", err)
```

Comment:
We should actually omit the "assuming external IP is not local" part and log it later when we check external IPs.

Reply:
It depends on above comments. If we permit returning partly correct addrs, it is hard to distinguish if addrs is correct or partly correct.